docker-hub/registry-access-management.md

```diff
@@ -0,0 +1,114 @@
+
+To configure Registry Access Management permissions, perform the following steps:
+
+1. Sign into your [Docker Hub](https://hub.docker.com) account as an organization owner.
+2. Select an organization, navigate to the **Settings** tab on the **Organizations** page and click **Org Permissions**.
+3. Enable Registry Access Management to set the permissions for your registry.
+
+> **Note**
+>
+> When enabled, the Docker Hub registry is set by default, however you can also restrict this registry for your developers.
+
+4. Click **Add** and enter your registry details in the applicable fields, and click **Create** to add the registry to your list.
+5. Verify that the registry appears in your list and click **Save & Apply**.  You can verify that your changes are saved in the Activity tab.
+
+  > **Note**
+  >
+  > Once you add a registry, it can take a maximum of 24 hours for the changes to be enforced on your developers’ machines. If you want to apply the changes sooner, you must force a Docker logout on your developers’ machine and have the developer re-authenticate for Docker Desktop.  Also, there is no limit on the number of registries you can add. See the Caveats sections to learn more about limitations  when using this feature.
+
+### Enforce authentication
+
+To ensure that each org member uses Registry Access Management on their local machine, you can perform the following steps below to enforce sign-in under your organization. To do this:
+
+1. Download the latest version of Docker Desktop, and then
+2. Create a `registry.json` file.
+
+Download Docker Desktop 4.5 or a later release.
+
+- [Download and install for Windows](/desktop/windows/install/)
+- [Download and install for Mac](/desktop/mac/install/)
+
+#### Create a registry json file
+
+Before you create a `registry.json` file, ensure that the developer is a member of at least one organization in Docker Hub. If the registry.json file matches at least one organization the developer is a member of, they can sign into Docker Desktop and access all of their organizations.
+
+**On Windows**
+
+On Windows, you must create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (that is, only the system administrator can write to the file). The file must be `JSON` and contain one or more organization names in the `allowedOrgs` key.
+
+To create your `registry.json` file on Windows:
+
+1. Open Windows Powershell and select Run as Administrator.
+2. Type the following command: `cd /ProgramData/DockerDesktop/`
+3. In Notepad, type `registry.json` and enter the Docker Hub organization that the user belongs to in `allowedOrgs` key and click Save.
+
+    For example:
+
+    ```json
+    {
+    "allowedOrgs": ["myorg"]
+    }
+    ```
+
+4. Navigate to Powershell and type ```start .```
+
+Congratulations! You have just created the registry.json file.
```

Comment:
It's not that big a deal! I suggest just removing this line.

Reply:
Updated